### PR TITLE
[zh-cn] Improve network-plugins page

### PR DESCRIPTION
**Pull request by mengjiao-liu**
When I was reading the network plugin page, I found this translation difficult to understand, which was a little different from the original English meaning.

page: https://kubernetes.io/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/

The difference is not very obvious, but when I read it for the first time, I really couldn't understand it. I only knew the meaning after reading the original English text.

Preview page: https://deploy-preview-35869--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/

### CODE REVIEW

File: content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ weight: 10
 Kubernetes {{< skew currentVersion >}} supports [Container Network Interface](https://github.com/containernetworking/cni)
 (CNI) plugins for cluster networking. You must use a CNI plugin that is compatible with your cluster and that suits your needs. Different plugins are available (both open- and closed- source) in the wider Kubernetes ecosystem. 
 -->
-Kubernetes {{< skew currentVersion >}} 支持[容器网络接口](https://github.com/containernetworking/cni) (CNI) 集群网络插件。
+Kubernetes {{< skew currentVersion >}} 支持用于集群网络的[容器网络接口](https://github.com/containernetworking/cni) (CNI) 插件。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md:18]: cluster networking ... 这里的 networking 是动词的名词化，指的是建立网络联接的工作，因此准确的翻译应该不是“集群网络”，而是“集群联网”。

Comment (mengjiao-liu) [reply content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md]: 好的，已更新。

**Review by ydFu — APPROVED** [review]
I also think this description is easier to understand.
/lgtm

**Pull request merged**